.github/workflows/codebuild-ci.yml

```diff
@@ -139,6 +139,20 @@ jobs:
         uses: aws-actions/aws-codebuild-run-build@v1
         with:
           project-name: aws-sdk-java-v2-endpoints-test
+  brazil-json-validation:
```

Comment:
Do we actually need code build job? Can we just use GH CI?

Reply:
Only advantage is we still need to build the project, so being on codebuild lets us keep the build time low by using larger instances.